README.md

```diff
@@ -104,7 +100,7 @@ Once the server is running, you can make a POST request with:
     {
         "query": "9887103",
         "documents": ["9887103", "30049242", "22936248"],
```

Comment:
```suggestion
        "documents": [30049242, 22936248],
```

Reply:
I think that is an example of the request right? not the response?

Reply:
Whoops, you're right